anaconda.py

```diff
@@ -291,16 +291,16 @@ def start_debugger(signum, frame):
         flags.dirInstall = True
 
     # Set up logging as early as possible.
-    import logging
-    from pyanaconda import anaconda_log
-    anaconda_log.init()
-    anaconda_log.logger.setupVirtio()
+    from pyanaconda import anaconda_logging
```

Comment:
Check the header of the `anaconda_loggers` module[0] - it's to avoid circular imports. The `anaconda_logging` module imports the `flags` module and the `flags` module also does logging, so it would have to import the `get_module_logger()` function from the `anaconda_logging` module, creating an import circle.

It's likely there could be other such circles, especially once we account for all the specialized loggers the `anaconda_loggers` module now provides getters for.

[0] https://github.com/rhinstaller/anaconda/pull/1041/files#diff-5c8919b8315eecb9a03a32af4f2bc02eR20